ui/src/components/settings/sub_pages/licenses.rs

```diff
@@ -0,0 +1,31 @@
+use common::icons::outline::Shape as Icon;
+use dioxus::prelude::*;
+use kit::elements::button::Button;
+use kit::elements::Appearance;
+use warp::logging::tracing::log;
+
+use crate::components::settings::SettingSection;
+
+#[allow(non_snake_case)]
+pub fn Licenses(cx: Scope) -> Element {
+    log::trace!("Licenses settings page rendered.");
+
+    cx.render(rsx!(
+        div {
+            id: "settings-licenses",
+            aria_label: "settings-licenses",
+            SettingSection {
+                section_label: "heroicons".into(),
+                section_description: "We have expanded upon the heroicons library we offer any additional icons under the same license as the original author.".into(),
+                Button {
+                    text: "MIT".into(),
+                    appearance: Appearance::Secondary,
+                    icon: Icon::DocumentText,
+                    onpress: move |_| {
+                        let _ = open::that("https://github.com/tailwindlabs/heroicons/blob/master/LICENSE");
```

Comment:
I thought we used `dioxus-heroicons`? If so or if its a forked of, it shouldnt point to tailwing but instead to dioxus-heroicons licenses instead, which would be a dual MIT and Apache 2.0 (The test could be `MIT/Apache 2.0" to represent dual license)

Reply:
We do, but the icons generated are not hosted within the repo usually, they are built from directories containing svg files, which we have moved into our repo but are originally from tailwindlabs project. No matter what bundling tool we're using I figure it boils down to being their icons.